docs/index.rst

```diff
@@ -48,6 +48,14 @@ A Modbus south service is added in the same way as any other south service in Fl
 
     - **Slave ID**: The slave ID of the Modbus device from which you wish to pull data.
 
+    - **Read Method**: The reading method defines the way to registers are read.
+
+      +------------+
+      | |modbus_3| |
+      +------------+
```

Comment:
Undefined substitution referenced of "modbus_3".

Reply:
It's fixed in https://github.com/fledge-iot/fledge-south-modbus-c/pull/49